core/src/main/kotlin/xtdb/api/log/LocalLog.kt

```diff
@@ -116,9 +116,7 @@ class LocalLog(rootPath: Path, private val instantSource: InstantSource, overrid
         try {
             val res = Array(msgs.size) { idx ->
                 val (msg) = msgs[idx]
-                // we only use the instantSource for Tx messages so that the tests
-                // that check files can be deterministic
-                val ts = if (msg is Message.Tx) instantSource.instant() else Instant.now()
+                val ts =  instantSource.instant()
```

Comment:
This is what creates most test file changes and might also confuse other people testing, but it makes my particular test scenario easier.